flexget/plugins/list/sonarr_list.py

```diff
@@ -26,7 +26,10 @@ class SonarrSet(MutableSet):
             'api_key': {'type': 'string'},
             'include_ended': {'type': 'boolean', 'default': True},
             'only_monitored': {'type': 'boolean', 'default': True},
-            'include_data': {'type': 'boolean', 'default': False}
+            'include_data': {'type': 'boolean', 'default': False},
+            'searchForMissingEpisodes': {'type': 'boolean', 'default': True},
```

Comment:
I know you wanted to stay true with the original Sonarr API name, but we use camel case where relevant. Please switch to that.

Reply:
Is it good names?

            'search_missing_episodes':
            'ignore_episodes_without_files'
            'ignore_episodes_with_files'

Reply:
kinda long but i dont have any better ideas